bemb/model/bemb.py

```diff
@@ -405,6 +413,41 @@ def sample_log_likelihoods(self, batch:ChoiceDataset, sample_dict: Dict[str, tor
 
         return log_likelihoods, log_likelihoods
 
+    @torch.no_grad()
+    def predict_proba(self, batch: ChoiceDataset) -> torch.Tensor:
+        """
+        Draw prediction on a given batch of dataset.
+
+        Args:
+        batch (ChoiceDataset): the dataset to draw inference on.
+
+        Returns:
+        torch.Tensor: the predicted probabilities for each class, the behavior varies by self.pred_item.
+        (1: pred_item == True) While predicting items, the return tensor has shape (len(batch), num_items), out[i, j] is the predicted probability for choosing item j AMONG ALL ITEMS IN ITS CATEGORY in observation i. Please note that since probabilities are computed from within-category normalization, hence out.sum(dim=0) can be greater than 1 if there are multiple categories.
```

Comment:
may not be a big deal, but I have not seen this out.* format

Reply:
Yeah, this is just to provide users some ideas on what is returned.